test/test_helper.rb

```diff
@@ -9,7 +9,7 @@
 
 Rails.backtrace_cleaner.remove_silencers!
 
-Seedbank.seeds_root = File.expand_path('dummy/db/seeds', __FILE__)
+Seedbank.application_root = Pathname.new(File.expand_path('../dummy', __FILE__))
```

Comment:
I definitely prefer single quotes unless interpolation, it's a good signpost to look closer when there are double quotes.